src/modules/10-common/navigation/ModuleConfigurationScreen/ModuleDetailsSection/ModuleCarousel.module.scss

```
@@ -10,8 +10,22 @@
   border-radius: 8px;
   position: relative;
   overflow: hidden;
-  padding: var(--spacing-xxlarge);
+  padding: var(--spacing-medium) !important;
   display: flex;
+  justify-content: center;
+  align-items: center;
+  .lottieLayer {
+    width: 100%;
+    height: 100%;
+    background-color: var(--grey-0);
+    border-radius: 8px;
```

NIT: try and use defined variables wherever possible

```suggestion
    border-radius: var(--spacing-3);
```